### PR TITLE
Implemented original PhaseNet model and converted weights

### PR DESCRIPTION
**Pull request by yetinam**
As identified in #151 the previous implementation of PhaseNet in SeisBench was different from the original. This PR implements the original version and converts the original weights. The previous implementation of PhaseNet is kept around as PhaseNetLight for compatibility. This PR uses ideas from #155.

### CODE REVIEW

**Comment by JUNZHU-SEIS** [comment]
This is quite an exciting update!
I have retrained the new PhaseNet model with DiTing, a data set from China. The new PhaseNet model exhibits less false picks on continuous data.
My question on the architecture of your new model is why you offset the skip connections with the convoluted outputs when concatenating them in the up-sampling layers. Is this related to PyTorch features? Any reply to this question is much appreciated.

**Comment by yetinam** [comment]
Super interesting to see the retrained model and the differences to PhaseNetLight. That seems quite promising. I'll also go into more systematic tests over the next days.

Regarding the offsets, these were necessary to achieve binary compatibility with the original PhaseNet written in tensorflow. Padding works slightly differently in pytorch and tensorflow and this adjusts for it.

For the pretrained models, this is expected. I haven't uploaded any models yet because that would break current SeisBench installations. I've thought of a solution now and will fix this in the next days. Plus I still need to actually train those models but I'm on it.

**Comment by yetinam** [comment]
The cache issue should be fixed now to. For the fixed PhaseNet there is only the original weight available so far though because I still need to train the others.

**Comment by zhong-yy** [comment]
Hi, it seems that `sbm.PhaseNet.from_pretrained("original")` has a very bad performance. Are you sure it is the original weights?

**Comment by yetinam** [comment]
Hi @zhong-yy ,
thanks for the observation. I'm sure for the weights themselves but maybe something in the processing around it does not currently fit? Could you please open a new issue for this (for easier tracking) and provide more details on the problems? In particular, it would be really helpful to have some examples.

**Comment by zhong-yy** [comment]
@yetinam Thank you for your quick reply. I am sorry I may have misused the model. When I use it with ‘model.eval()’ the performance is bad. After I remove ‘model.eval()’ it works well.
But I don’t know when I should use model.eval(). I have opened a new issue about this question
